entries/entry04.md

```diff
@@ -1,8 +1,114 @@
 # Entry 4
```

Comment:
Sources: 1
EDP: 1
Knowledge: 1
Skills: 1
Professionalism: 1

Excellent job! I love how you included your tinkering in jsbin.